src/components/policy/policy_regular/src/update_status_manager.cc

```diff
@@ -116,6 +118,10 @@ void UpdateStatusManager::OnNewApplicationAdded(const DeviceConsent consent) {
     return;
   }
   app_registered_from_non_consented_device_ = false;
+  if (last_update_was_failed_) {
+    last_update_was_failed_ = false;
+    ProcessEvent(kPostponedUpdate);
```

Comment:
are you sure that we still need to call `ProcessEvent(kOnNewAppRegistered);` if we enter this condition?

Reply:
@mked-luxoft  `ProcessEvent(kPostponedUpdate) `added for extra call `UPDATE_NEEDED` for new ptu if last was finished with the same status. I tried remove `ProcessEvent(kOnNewAppRegistered)` but it entailed a lot of regressions